webapp/src/components/App/Singlestudy/HomeView/InformationView/CreateVariantDialog.tsx

```diff
@@ -36,14 +36,17 @@ interface Props {
 function CreateVariantDialog({ parentId, open, variantTree, onClose }: Props) {
   const [t] = useTranslation();
   const navigate = useNavigate();
-  const [sourceList, setSourceList] = useState<GenericInfo[]>([]);
-  const defaultValues = { name: "", sourceId: parentId };
 
-  const existingVariants = useMemo(() => sourceList.map((variant) => variant.name), [sourceList]);
+  const studiesSource = useMemo(() => createListFromTree(variantTree), [variantTree]);
+
+  const studySourceOptions = useMemo(
+    () => studiesSource.map(({ id, name }) => ({ value: id, label: name })),
+    [studiesSource],
+  );
```

Comment:
suggestion: 
`studiesSource` -> `variantList`

`studySourceOptions` -> `variantOptions`

Reply:
There is also the reference study in the list